src/api/controller/MyController.hpp

```diff
@@ -78,31 +65,31 @@ class MyController : public oatpp::web::server::api::ApiController {
 
     auto responseDto = bakcyl::api::LoginResponseDto::createShared();
 
-    if (json && json->username && json->password) {
-      std::string token = auth.authenticateUser(json->username, json->password);
-
-      bool loginSuccess = false;
-      if (token != "") loginSuccess = true;
-      responseDto->success = loginSuccess;
-
-      if (loginSuccess) {
-        auto response = createDtoResponse(Status::CODE_200, responseDto);
-        response->putHeader("Authorization", "Bearer " + token);
-        return response;
-      } else {
-        return createDtoResponse(Status::CODE_403, responseDto);
-      }
-    } else {
+    if (!json && !json->username && !json->password) {
       responseDto->success = false;
       return createDtoResponse(Status::CODE_400, responseDto);
     }
+
+    std::string token = auth.authenticateUser(json->username, json->password);
```

Comment:
const auto token

Reply:
the commit 20cf9ae solved the issue on #90